### ISSUE TITLE
explicit artworks aren't showing on homepage

### ISSUE THREAD

**Issue by Yuukinoo** [Notsfsssf/pixez-flutter#827]
update: i feel like this is an issue due to how pixiv work because this also happens within that site, feel free to delete this issue

**Comment by kyoyacchi** [comment]
I guess you need to enable R18 from pixiv settings first.(Enable it on app as well)
![Screenshot_20241112-180341](https://github.com/user-attachments/assets/228d3792-5fcf-4fc2-9e1f-fa242ce2c3a3)
https://www.pixiv.net/settings/viewing

Try changing location if you **CAN NOT**
https://www.pixiv.net/settings/language-and-location

**Comment by Yuukinoo** [comment]
yeah i'm aware that at least one of those need to be enabled so i already have r18 option on, my problem is they only show in "Related illusts" section while i'm currently looking at an r18 artwork (as shown) and not anywhere on the home page at all![Screenshot_20241117_023540.png](https://github.com/user-attachments/assets/bff260e0-8877-48e6-97bc-566a9e89b967)

.

**Comment by kyoyacchi** [comment]
> read above

bro thinks we can't see his  edit history 😭😭😭

**Comment by Yuukinoo** [comment]
it's not that i care about the previous content lol i just think it doesn't matter anymore since this is also apparently a problem with pixiv (with pixiv it's not as problematic because there's a toggle to show explicit art on homepage)
